var/exp_repo/experiments/scalingexperiment/experiment.py

```diff
@@ -0,0 +1,124 @@
+from benchpark.directives import variant
+
+
+class ScalingExperiment(object):
+    variant(
+        "scaling-factor",
+        default="2",
+        values=int,
+        description="Factor by which to scale values of problem variables",
+    )
+
+    variant(
+        "scaling-iterations",
+        default="4",
+        values=int,
+        description="Number of experiments to be generated",
+    )
+
+    # input parameters:
+    # 1. input_variables: dict[str, int | tuple(str), list[int]]. Input variables
+    # For the first value in input_variables, if the value is a list, select the
+    # index of its smallest element, 0 otherwise
+    # Beginning with this index, generate a list of indexes of length equal to
+    # the number of dimensions in an (ascending) round-robin order
+    #
+    # output:
+    # scaling_order: list[int]. list of num_exprs values, one for each dimension,
+    # starting with the minimum value of the first element in input_variables arranged
+    # in an ascending round-robin order
+    def configure_scaling_policy(self, input_variables):
+        # compute the number of dimensions
+        n_dims = 1
+        for param in input_variables.values():
+            if isinstance(param, list):
+                n_dims = len(param)
+                break
+
+        # starting with the minimum value dim of the ordering parameter
+        # compute the remaining n_dims-1 values in a round-robin manner
+        val = input_variables[next(iter(input_variables))]
+        min_dim = val.index(min(val)) if isinstance(val, list) else 0
+
+        return [(min_dim + i) % n_dims for i in range(n_dims)]
```

Comment:
(suggestion) Since this doesn't sort (e.g. doesn't make sure that the second-lowest dimension is handled second), IMO this would be simpler if it just returned `min_dim`.